seatunnel-e2e/seatunnel-connector-v2-e2e/connector-kafka-e2e/src/test/resources/kafka/kafkasource_earliest_to_console.conf

```
@@ -69,6 +69,16 @@ sink {
             ]
           }
         ]
+        row_rules = [
+                {
+                  rule_type = MIN_ROW
+                  rule_value = 1
```

```suggestion
                  rule_value = 100
```
set `MAX_ROW` and `MIN_ROW` as same.

---

i known,all test updated